l10n_th_partner/models/res_users.py

```diff
@@ -1,12 +1,15 @@
 # Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th/)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class Users(models.Model):
     _inherit = "res.users"
 
+    first_name = fields.Char(translate=True)
+    last_name = fields.Char(translate=True)
```

Comment:
Why do you have these fields ? In the core odoo use name is firstname and lastname.